docs/guides/columns-post-type.md

```diff
@@ -0,0 +1,76 @@
+---
+id: columns-post-type
+title: Post Type
+---
+
+Login class is located in `Eightshift Libs`. To extend it use `Eightshift_Libs\Columns\Base_Post_Type_Columns` class.
+
+## Example:
+
+To create custom taxonomy:
+1. make a new class `src/columns/class-example-column.php`
+2. extend all methods provided in the example.
+3. register the class inside `class-main.php` file.
+4. Dump autoload
```

Comment:
```suggestion
4. dump autoload
```

Either all the points start with lowercase or with an uppercase letter. I'm fine with both tbh